cypress/e2e/signIn.cy.js

```diff
@@ -12,23 +12,50 @@ describe('Sign In page', () => {
 
   before(() => {
     cy.task('db:clear');
```

Comment:
Ensure that the `db:clear` task is correctly set up in your Cypress tasks. This task should reset the database to a known state before each test to ensure test isolation and reliability.